src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -3282,4 +3286,107 @@ public Response getCurationStates() throws WrappedResponse {
         csvSB.append("\n");
         return ok(csvSB.toString(), MediaType.valueOf(FileUtil.MIME_TYPE_CSV), "datasets.status.csv");
     }
+    
+    //APIs to manage archival status
+    
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/submitDatasetVersionToArchive/{id}/{version}/status")
+    public Response getDatasetVersionToArchiveStatus(@PathParam("id") String dsid,
+            @PathParam("version") String versionNumber) {
+
+        try {
+            AuthenticatedUser au = findAuthenticatedUserOrDie();
+            if (!au.isSuperuser()) {
+                return error(Response.Status.FORBIDDEN, "Superusers only.");
+            }
+            Dataset ds = findDatasetOrDie(dsid);
+           
+            DatasetVersion dv = datasetversionService.findByFriendlyVersionNumber(ds.getId(), versionNumber);
+            if (dv.getArchivalCopyLocation() == null) {
+                return error(Status.NO_CONTENT, "This dataset version has not been archived");
+            } else {
+                JsonObject status = JsonUtil.getJsonObject(dv.getArchivalCopyLocation());
+                return ok(status);
+            }
+        } catch (WrappedResponse wr) {
+            return wr.getResponse();
+        }
+    }
+
+    @PUT
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/submitDatasetVersionToArchive/{id}/{version}/status")
+    public Response setDatasetVersionToArchiveStatus(@PathParam("id") String dsid,
+            @PathParam("version") String versionNumber, JsonObject update) {
+
+        logger.info(JsonUtil.prettyPrint(update));
```

Comment:
I agree. (FWIW - these are at logger.info because the DataCommons dev machine was deployed with payara 2021.5 which has the trouble with set-log-levels.)

Reply:
Fixed.